packages/api-nest/src/repositories/interfaces/highlight-repository.interface.ts

```
@@ -0,0 +1,47 @@
+   */
+  findById(id: string, userId: string): Promise<HighlightEntity | null>
+
+  /**
+   * Find all highlights for a library item
+   * @param libraryItemId - Library item ID
+   * @param userId - User ID who owns the highlights
+   * @returns Array of highlights, sorted by position
+   */
+  findByLibraryItem(
+    libraryItemId: string,
+    userId: string,
```

ESLint

[error] Delete `,`

(prettier/prettier)